exchange-application/frontend/v3.1/src/FetchUtils.js

```diff
@@ -0,0 +1,26 @@
+async function fetchWithRetry (url, options=null, retries=3, msTimeout=3000) {
+    const timerPromise = createTimerPromise(msTimeout)
+    try {
+        const response = await Promise.race([fetch(url, options), timerPromise]);
+        if (!response.ok && process.env.REACT_APP_CATCH_HTTP_ERRORS) {
```

Comment:
Here's the only change. I added the env variable to the condition

Reply:
IMHO kind of a strange name for the variable. I'd prefer something like `REACT_APP_ENABLE_RETRY` or something similar. 

But it's a nitpick, adding +1.

Reply:
@m-czernek  What about `REACT_APP_ENABLE_RETRY_INVALID_REQUESTS`? REACT_APP_ENABLE_RETRY` is too general and it doesn't affect retries on timeouts or network failures.

Reply:
Sure, that works better IMO.

Reply:
I changed the variable name to `REACT_APP_RETRY_INVALID_REQUESTS`. Thanks @m-czernek !